### PR TITLE
CSS Transition to notify__popup

### LINKED ISSUE #300: Notify jumps on resize
Original issue https://community.adaptlearning.org/mod/forum/discuss.php?d=4078

### PR DESCRIPTION
**Pull request by HenrikAabom**
fixes #300 

Created CSS transition for notify to smooth out the jumps it makes when it contains an article that can resize via an Accordion

### CODE REVIEW

**Comment by oliverfoster** [comment]
Thanks @HenrikAabom 

We always have an associate issue with prs. I've made one for you, #300 and invited someone to review it.

**Comment by HenrikAabom** [comment]
Thanks @oliverfoster 
I didn't know how to create an issue :)

**Comment by oliverfoster** [comment]
:+1: Just because I love drawing arrows on screen shots:

image

**Comment by HenrikAabom** [comment]
Haha! Thanks :D

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by oliverfoster** [comment]
@guywillis

**Review by kirsty-hames — COMMENTED** [review]
Just for consistency should this use the transition mixin? Fix looks good.

e.g. `transition(margin-top 500ms ease-in, top 500ms ease-in;);`

**Comment by kirsty-hames** [comment]
> @kirsty-hames I couldn't find the transition mixin in the theme. It broke when I switched it over.

Hey @oliverfoster, the `.transition` mixin is in core https://github.com/adaptlearning/adapt-contrib-core/blob/57d64c30865faf06c82711fb5ac1bedc4af36f96/less/core/transitions.less

I think I'd have to take a look to see where it's breaking.

File: less/core/notify.less [hunk 1]
```diff
@@ -2,6 +2,7 @@
   &__popup {
     background-color: @notify;
     color: @notify-inverted;
+    transition: margin-top 500ms, top 500ms;
```
Comment (oliverfoster) [inline less/core/notify.less:5]:
```suggestion
    .transition(margin-top 500ms, top 500ms;);
```

**Review by ethan-lp — APPROVED** [review]

**Comment by oliverfoster** [comment]
@HenrikAabom could you make the required changes please?

**Pull request merged**